frame/staking/src/lib.rs

```diff
@@ -910,6 +910,34 @@ decl_module! {
 			}
 		}
 
+		/// Rebond a portion of the stash scheduled to be unlocked.
+		///
+		/// # <weight>
+		/// - Insignificant complexity, bounded by `MAX_UNLOCKING_CHUNKS`.
+		/// - Can't increase storage, only decrease it.
+		/// # </weight>
+		#[weight = SimpleDispatchInfo::FixedNormal(500_000)]
+		fn rebond(origin, #[compact] value: BalanceOf<T>) {
+			let controller = ensure_signed(origin)?;
+			let mut ledger = Self::ledger(&controller).ok_or(Error::<T>::NotController)?;
```

Comment:
You could check `ledger.unlocking.len()` here and short circuit if there hasn't been any pending funds to unbond.